src/spetlr/etl/loaders/scd2_loader.py

```diff
@@ -0,0 +1,116 @@
+from typing import List
+
+import pyspark.sql.functions as f
+from pyspark.sql import DataFrame
+
+from spetlr.delta import DeltaHandle
+from spetlr.etl import Loader
+from spetlr.functions import get_unique_tempview_name
+from spetlr.transformers import ValidFromToTransformer
+from spetlr.utils import GetMergeStatement
+
+
+class SCD2Loader(Loader):
```

Comment:
Another name? ValidFromToUpsertLoader? @lasseaj  what do you think? SCD2 is a very more broad term. This name change suggestion also lies closer to the transformer.

Reply:
If we plan on adding a more slowly changing dimension components, then i think a name such as `SCD2UpsertLoader` would be really neat. We could also just add it as an alias, as in 
```python
class ValidFromToUpsertLoader(Loader):
    ... # implementation here

SCD2UpsertLoader = ValidFromToUpsertLoader
```

Reply:
Good idea. I have done that!